atom/common/promise_util.h

```diff
@@ -46,6 +46,16 @@ class Promise : public base::RefCounted<Promise> {
     return GetInner()->Reject(GetContext(), v8::Undefined(isolate()));
   }
 
+  v8::MaybeLocal<v8::Promise> Then() {
```

Comment:
This would `CHECK` crash right? `v8::Undefined` can't be cast to `v8::Function`.

I don't think a zero-param `Then` is needed here